NickvisionTagger.Shared/Helpers/FfprobeHelpers.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Text.Json;
+
+public class FfprobeFormat
+{
+    public string Filename { get; set; }
+    public Dictionary<string, string> Tags { get; set; }
+
+    public FfprobeFormat()
+    {
+        Filename = "";
+        Tags = new Dictionary<string, string>();
+    }
+    
+    public override string ToString()
+    {
+        var s = $"==={Filename}===\n";
+        foreach (var pair in Tags)
+        {
+            s += $"{pair.Key} - {pair.Value}\n";
+        }
+        s = s.Remove(s.Length - 1, 1);
+        return s;
+    }
+}
+
+public static class FfprobeHelpers
+{
+    public static FfprobeFormat? GetFormat(string ffprobePath, string filePath)
+    {
+        using var process = new Process()
+        {
+            StartInfo = new ProcessStartInfo()
+            {
+                FileName = ffprobePath,
+                Arguments = $"\"{filePath}\" -print_format json -v quiet -show_format",
```

Comment:
That works for flac, wav and mp3, but not for ogg and opus. For them you need `-show_streams` and in case of ogg you also need to detect which stream is audio (because there will be video stream with cover even for audio-only file).

Reply:
I tested with an ogg file and this worked no problem without show streams...I see why we'd need to check audio only stream, but shouldn't that be done at MusicFile construction to begin with, to make sure it's actually a music (audio) file?

Reply:
I have to check when I get back to my computer but I was getting correct results for ogg no problem...

Reply:
I think as universal solution you can use both `-show_format -show_streams` and then look for tags in `format` first, then in each `stream` with `codec_type: audio`, and this should work for any container.

Reply:
> I think as universal solution you can use both `-show_format -show_streams` and then look for tags in `format` first, then in each `stream` with `codec_type: audio`, and this should work for any container.

That sounds like a good idea...will do :)